lib/happo/public/happo-styles.css

```diff
@@ -8,4 +8,27 @@ body {
     #f4f4f4 10px,
     #f4f4f4 20px
   );
+  margin: 0;
+}
+
+header {
```

Comment:
I know we aren't dealing with many pages, but I think these styles would be a lot better if we used BEM and avoided using element selectors entirely.